Airports/EGCC/SMR/Regions.txt

```diff
@@ -109,7 +113,8 @@ N053.20.39.462 W002.16.54.370
 N053.20.48.213 W002.16.36.440
 ;outer_taxiways
 REGIONNAME Manchester
-smrLLOutline N053.19.55.957 W002.18.39.581
+smrLLOutline N053.19.58.583 W002.18.43.162
```

Comment:
This colour used to be smrCCborder - can you work out why it was changed?

Reply:
No clue at all - that's how I found it!

Reply:
And it is not only that... Some regions are named completely wrong!
![default](https://user-images.githubusercontent.com/18355854/44336924-a45b8d80-a481-11e8-8bc9-883ed7efd59d.png)

Reply:
@hsugden Found this: https://github.com/tasosb/UK-Sector-File/commit/03203f6fef7c3aaf1f3c2c992201322f3b955fc3#diff-940dbfc14826345b690b2af620a8fcac
Seems that @cpawley changed it back then.

Reply:
and that's how it looks with smrCCBorder 
![default](https://user-images.githubusercontent.com/18355854/44494879-4a6fe900-a676-11e8-8ebd-4a27bf560906.png)